### ISSUE TITLE
[bitnami/postgresql] 0.7.1 version does not exist in bitnami chart repo, but only in deprecated-stable chart repo.

### ISSUE THREAD

**Issue by dosuken123** [bitnami/charts#4050]
**Which chart**:
bitnami/postgresql

**Is your feature request related to a problem? Please describe.**
GitLab has a feature called [Auto Deploy](https://docs.gitlab.com/ce/topics/autodevops/stages.html#auto-deploy), which depends on [a Helm chart](https://gitlab.com/gitlab-org/cluster-integration/auto-deploy-image/-/tree/master/assets/auto-deploy-app) to deploy user's application.

This feature currently uses [`stable/postgresql` chart](https://gitlab.com/gitlab-org/cluster-integration/auto-deploy-image/-/blob/master/src/bin/auto-deploy#L183). For the newer installations, [v8.2.1](https://gitlab.com/gitlab-org/cluster-integration/auto-deploy-image/-/blob/master/src/bin/auto-deploy#L175) is used. For the legacy installations [v0.7.1](https://gitlab.com/gitlab-org/cluster-integration/auto-deploy-image/-/blob/master/assets/auto-deploy-app/requirements.yaml#L3) is used.

Now, as you may already notice, [the stable chart repository will be gone by Nov 13, 2020](https://github.com/helm/charts/issues/23742#issuecomment-702128318). Hence, In GitLab, we [decided to move to the `bitnami/postgresql`](https://gitlab.com/gitlab-org/gitlab/-/issues/263778) chart as it's new standard after the deprecation/removal. However, [`v0.7.1` does NOT exist in the `bitnami/postgresql` chart](https://gitlab.com/gitlab-org/gitlab/-/issues/263778#note_430792503). Hence, we cannot clearly migrate to bitnami's chart.

Can v0.7.1 chart be released in `bitnami/postgresql`? I see a couple of old versions are available currently, for example,

```
shinya@shinya-MS-7A34:~/workspace/auto-deploy-image$ helm search repo bitnami/postgresql -l | grep "\s0."
bitnami/postgresql      0.4.25          10.4.0          Chart for PostgreSQL                              
bitnami/postgresql      0.4.24          10.3.0          Chart for PostgreSQL                              
bitnami/postgresql      0.4.23          10.3.0          Chart for PostgreSQL                              
bitnami/postgresql      0.4.22          10.3.0          Chart for PostgreSQL                              
bitnami/postgresql      0.4.21          10.3.0          Chart for PostgreSQL                              
bitnami/postgresql      0.4.20          10.3.0          Chart for PostgreSQL                              
bitnami/postgresql      0.4.19          10.2.0          Chart for PostgreSQL                              
bitnami/postgresql      0.4.18          10.2.0          Chart for PostgreSQL                              
bitnami/postgresql      0.4.17          10.1.0          Chart for PostgreSQL                              
bitnami/postgresql      0.4.16          10.1.0          Chart for PostgreSQL                              
bitnami/postgresql      0.4.15          10.0.0          Chart for PostgreSQL                              
bitnami/postgresql      0.4.14          9.6.5           Chart for PostgreSQL                              
bitnami/postgresql-ha   0.4.0           11.5.0          Chart for PostgreSQL with HA architecture (usin...
```

So I think it makes sense to add more legacy chart. Does it make sense?

**Describe the solution you'd like**
Add v0.7.1 chart (which is identical to `stable/postgresql`'s v0.7.1) to `bitnami/postgresql`.

**Describe alternatives you've considered**
If the above solution cannot be accepted, we'll [vendor the `stable/postgresql` in our repository](https://gitlab.com/gitlab-org/cluster-integration/auto-deploy-image/-/merge_requests/127).

**Additional context**
Since this is time-sensitive task (we need to make a decision by two/three weeks prior to the removal date), I'd appreciate if you'd respond the above question as soon as possible. Thank you for your help!

**Comment by dosuken123** [comment]
cc @juan131

**Comment by carrodher** [comment]
Hi, I am sorry but the `stable/postgresql` version 0.7.1 (1 Jun 2017) was not maintained by Bitnami, see [the `maintainers` section in the _Chart.yaml_ file](https://github.com/helm/charts/blob/b90ad657e1a226eb52c3eb6a2a95ba3d6d494f58/stable/postgresql/Chart.yaml#L14). 

At this date, Bitnami has its own `bitnami/postgresql` Helm Chart hosted in the `bitnami` repository but without any relation to the one hosted in `stable`.

It was in `stable/postgresql` version 2.0.0 (18 Jan 2019) when the bitnami and stable charts were merged and from this moment the Bitnami team maintains the `stable/postgresql` chart, see https://github.com/wgiddens/charts/commit/2767f194dc6bfc13e395beb8f8c815024b0b0393. So this 2.0.0 version is the minimal one that is going to be the same in the `stable` and `bitnami` repository.

We already worked with the GitLab team in the past to use the Bitnami Helm Charts as dependencies for the GitLab one, we are happy to help if needed to migrate to a more recent version that can be found in the Bitnami repository, but as I mention, the `stable/postgresql` 0.7.1 was a different chart no maintained by us so we can't push it to our registry.
